### PR TITLE
Use existing left-alignment feature for dashboard-driven dialogs.

### PR DESCRIPTION
**Pull request by breville**
It appears that none of them feature an icon.
I couldn't find a use of pre_image in any of our levels.
This made the "unindent" alignment unnecessary.

### CODE REVIEW

**Comment by breville** [comment]
@davidsbailey interested?  anybody else?

File: dashboard/app/views/levels/_dialog.html.haml [hunk 1]
```diff
@@ -28,17 +28,10 @@
           =t('dialog.ok')
 
   #pre-dialogcontent
-    - if data['pre_image']
```
Comment (davidsbailey) [inline dashboard/app/views/levels/_dialog.html.haml]: Please make sure this works correctly a match level which has a pre_image:
```
Davids-MacBook-Pro-2:~/src/cdo/dashboard$ grep -R -l -w pre_image * | grep match
config/scripts/2-3/maze/2_3_maze_match_2.match
config/scripts/4-5/maze/4-5_maze_match_1.match
config/scripts/k-1/Maze/k-1_maze_match.match
config/scripts/test/matches/test_matching_image.match
```

**Comment by davidsbailey** [comment]
LGTM. Heads up that this might break a few eyes tests.

**Comment by breville** [comment]
Thanks @davidsbailey .. pre_image really didn't seem to be used so I completely removed support and all mentions of it.

**Pull request merged**